add an FAQ entry about zaraz.ecommerce (#5853)
cloudflare/cloudflare-docs: content/zaraz/faq.md

### I am a vendor. How can I integrate my tool with Zaraz?

The Zaraz team is working with third-party vendors to build their own Zaraz integrations using the Zaraz SDK. To request a new tool integration, or to collaborate on our SDK, contact us at zaraz@cloudflare.com.

> ### Using the zaraz.ecommerce() method returns undefined error
> 
> Zaraz E=commerce features need to be enabled in [the Zaraz Settings page](https://developers.cloudflare.com/zaraz/reference/settings/#e-commerce-tracking). Make sure you enabled E-commerce tracking before trying to use the Web API. 

Reviewer: ```suggestion
### Using the `zaraz.ecommerce()` method returns an undefined error.

E-commerce tracking needs to be enabled in [the Zaraz Settings page](https://developers.cloudflare.com/zaraz/reference/settings/#e-commerce-tracking) before you can start using the E-commerce Web API.
```
